src/sentry/static/sentry/app/views/dataExport/dataDownload.tsx

```diff
@@ -160,10 +208,11 @@ class DataDownload extends AsyncView<Props, State> {
             <br />
             {this.renderDate(dateExpired)}
           </p>
-          <small>
-            <strong>SHA1:{checksum}</strong>
-          </small>
+          {this.renderOpenInDiscover()}
           <p>
+            <small>
+              <strong>SHA1:{checksum}</strong>
+            </small>
             {tct('Need help verifying? [link].', {
```

Comment:
In your screenshot there isn't space between the checksum and 'Need help'. It looked better before when there was a line break.

Reply:
whoops, let me fix that